app/src/main/res/values/preference_keys.xml

```diff
@@ -150,10 +150,10 @@
     <string name="preference_key_edit_line_numbers">editLineNumbers</string>
     <string name="preference_key_edit_typing_suggestions">editTypingSuggestions</string>
     <string name="preference_key_reading_lists_share_url_shorten">useUrlShortenerForSharing</string>
-    <string name="preference_key_reading_lists_share_survey_attempts">readingListShareSurveyAttempts</string>
+    <string name="preference_key_reading_lists_share_survey_dialog_shown">readingListShareSurveyAttempts</string>
```

Comment:
Should we update the string value to `readingListShareSurveyDialogShown`?

Reply:
> Should we update the string value to `readingListShareSurveyDialogShown`?

Just verified that the value should be updated since the type is different and it will cause a crash when upgrading to new version.

Reply:
Thank you Cooltey... good catch!